backend/src/fetcher.py

```diff
@@ -160,14 +160,27 @@ def fetch_spec_file(self) -> Optional[dict[str, str]]:
         return {"name": spec_name, "content": response.text}
 
 
+def find_matching_koji_subtask(
```

Comment:
I'm not sure I understand how is this related to the standalone scripts this PR is introducing?

Reply:
I reused the backend logic in the new script: https://github.com/fedora-copr/log-detective-website/pull/114/files#diff-816f1b55c694752aa1794eb4c879b417609a56727a3b902d0b11b2960ea3a4ecR19